docs/Reading.md

```diff
@@ -97,3 +97,23 @@ while (logicalColumnReader.HasNext)
 
 The .NET type used to represent read values can optionally be overridden by using the `ColumnReader.LogicalReaderOverride<TElement>` method.
 For more details, see the [type factories documentation](TypeFactories.md).
+
+## DateTimeKind when reading Timestamp's
+When reading Timestamp's to an DateTime, ParquetSharp will set the DateTimeKind based on the value of `IsAdjustedToUtc`.
+
+If `IsAdjustedToUtc` is `true` the DateTimeKind will be set to `DateTimeKind.Utc` otherwise it will be set to `DateTimeKind.Unspecified`.
+
+This behavior can be overwritten by setting the AppContext switch `ParquetSharp.UseDateTimeKindUnspecified` to `true`.  
+This also matches the old behavior of [ParquetSharp > 6.0.0](https://github.com/G-Research/ParquetSharp/pull/261)
```

Comment:
Shouldn't it be less than version 7.0.0?
```suggestion
This also matches the old behavior of [ParquetSharp < 7.0.0](https://github.com/G-Research/ParquetSharp/pull/261)
```

Reply:
Your right! I messed up the sign and [version](https://github.com/G-Research/ParquetSharp/releases/tag/7.0.0).